src/style.css

```diff
@@ -0,0 +1,119 @@
+@font-face {
+  font-family: "Avenir";
+  font-weight: 300;
+  font-style: normal;
+  src: url("./fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 600;
+  font-style: normal;
+  src: url("./fonts/Avenir-Heavy.ttf");
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+  font-weight: 300;
+}
+
+.form-field {
+  display: flex;
+  margin-top: 20px;
+  width: 100%;
+}
+
+.form-field__big-bar {
+  width: 100%;
+  height: 70px;
+}
+
+.form-field__small-bar {
+  width: 100%;
+  height: 42px;
+}
+
+.form-field__lable {
+  position: relative;
+  display: block;
+  width: 100%;
+}
+
+.form-field__lable:after {
+  position: absolute;
+  content: "";
+  display: block;
+}
+
+.form-field__lable-small:after {
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+  background-size: contain;
+}
+
+.form-field__lable-big:after {
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
+  background-size: contain;
+}
+
+.form-field__typing {
+  padding: 0;
+  width: 100%;
+  height: 100%;
+  border-radius: 4px;
+  outline: none;
+  border: 1px solid #e1e7ed;
+  color: #3d4e61;
+  font-weight: inherit;
+  font-family: inherit;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+  line-height: 19px;
+  position: relative;
+}
+
+.form-field__typing::placeholder {
+  color: #3d4e61;
+  font-weight: 300;
+}
+
+.form-field__typing-small {
+  padding-left: 33px;
+  font-size: 14px;
+}
+
+.form-field__typing-small::placeholder {
+  font-size: 14px;
+}
+
+.form-field__typing-big {
+  padding-left: 62px;
+  font-size: 16px;
+}
+
+.form-field__typing-big::placeholder {
+  font-size: 16px;
+}
+
+.form-field__typing:hover {
+  box-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.25);
+}
+
+.form-field__typing:focus {
+  border-radius: 3px;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  font-weight: 600;
```

Comment:
Check mockup font-weight

Reply:
the font-weight is always 400, but for focus I think that it is a mistake because it should be 900 for both small and big input